index.js

```diff
@@ -213,6 +213,11 @@ class ReadableState {
     this.map = mapReadable || map
     this.pipeTo = null
     this.afterRead = afterRead.bind(this)
+    if (signal) {
+      signal.addEventListener('abort', () =>
```

Comment:
signal.addEventListener('abort', this.destroy.bind(this, new Error('Stream aborted'))

Reply:
Creating the error instance in the constructor will have an Error stack for the constructor while having a fat arrow function will have an Error stack revealing the location where `controller.abort` is called which might help when seeing this error. May be helpful?

Reply:
Then I think we should make a function abort () { this.destroy(...) } and bind that

Reply:
But it should live in the stream constructor still so we have it everywhere. That’s where the destroy method lives as well